File: common/src/main/java/com/wdz/common/net/response/ProjectResponse.java
package com.wdz.common.net.response;

import java.util.List;

public class ProjectResponse {
    public List<String> children;
    public int courseId;
    public int id;
    public String name;
    public int order;
    public int parentChapterId;
    public boolean userControlSetTop;
    public int visible;

    @Override
    public String toString() {
        return "Data{" +
                "children=" + children +
                ", courseId=" + courseId +
                ", id=" + id +
                ", name='" + name + '\'' +
                ", order=" + order +
                ", parentChapterId=" + parentChapterId +
                ", userControlSetTop=" + userControlSetTop +
                ", visible=" + visible +
                '}';
    }
}

File: kt_common/src/main/java/com/wdz/ktcommon/http/response/ProjectResponse.kt
package com.wdz.ktcommon.http.response

data class ProjectResponse(var children: List<String>,
                           var courseId: Int,
                           var id: Int,
                           var name: String,
                           var order: Int,
                           var parentChapterId: Int,
                           var userControlSetTop: Boolean,
                           var visible: Int) {
}